src/entryPoints/SidebarValues.res

```diff
@@ -331,8 +331,17 @@ let businessProfiles = () => {
     searchOptions: [("Configure business profiles", "")],
   })
 }
+
+let configurePMTs = permissionJson => {
+  SubLevelLink({
+    name: "Configure PMTs",
+    link: `/configure-pmts`,
+    access: permissionJson.connectorsView,
+    searchOptions: [("Configure payment methods", "")],
```

Comment:
Can we add more search params like `add country `, `add currency` ,`configure country currency`